core/models.py

```diff
@@ -59,3 +59,29 @@ def save(self, *args, **kwargs):
             self.content_type = self.content_type.decode("utf-8")
 
         super().save(*args, **kwargs)
+
+
+class SiteSettings(models.Model):
+    wordcloud_ignore = models.TextField(
+        default="",
+        help_text="A comma-separated list of words to ignore in the release report wordcloud.",  # noqa
+    )
+
+    class Meta:
+        constraints = [
+            # check constraint to only allow id=1 to exist
+            models.CheckConstraint(
```

Comment:
Good call putting this in a CheckConstraint rather than trying to cover all methods that write to the db.